eth/watchers/timewatcher.go

```diff
@@ -75,11 +76,18 @@ func (tw *TimeWatcher) LastInitializedBlockHash() [32]byte {
 	return tw.lastInitializedBlockHash
 }
 
-func (tw *TimeWatcher) setLastInitializedRound(round *big.Int, hash [32]byte) {
+func (tw *TimeWatcher) CurrentRoundStartBlock() *big.Int {
+	tw.mu.RLock()
+	defer tw.mu.RUnlock()
+	return tw.currentRoundStartBlock
+}
+
+func (tw *TimeWatcher) setLastInitializedRound(round *big.Int, hash [32]byte, num *big.Int) {
```

Comment:
Is there a more descriptive name for the `num` param? `startBlk`?

Reply:
fixed in 70f0e9d